### PR TITLE
SB-62-hot-fixes

### PR DESCRIPTION
**Pull request by StudentCZ**
Found a minor error with GET AND POST Routes when https://github.com/BFT-Smartbus/smartbus-heartbeat/pull/13 got merged to main.

Need to change this to this
Screen Shot 2023-01-05 at 10 27 33 PM

To This because DynamoDB has it as a string and not integer
Screen Shot 2023-01-05 at 10 27 38 PM

Also this
Screen Shot 2023-01-05 at 10 27 58 PM
To This because we are querying UserId in the DB and user_id don't exist
Screen Shot 2023-01-05 at 10 28 05 PM

Also refractor the constants to keep consistency in all 3 py files.
Screen Shot 2023-01-05 at 10 12 25 PM
Screen Shot 2023-01-05 at 10 13 16 PM
Screen Shot 2023-01-05 at 10 33 03 PM

ADDED TRY EXCEPT TO TABLE
Screen Shot 2023-01-05 at 10 12 43 PM

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 from flask import Flask, request, jsonify
 from flask_cors import CORS, cross_origin
+SETTINGS = 'settings.yaml'
+MAX_LOOKBACK = 10
 
 app = Flask(__name__)
 CORS(app)
 
-MAX_LOOKBACK = 10
+with open(SETTINGS, 'r') as yaml_file:
```
Comment (yhus) [inline app.py:14]: To take this one step further, you could factor all of this logic into a dedicated settings.py

This way, all the files/modules in this repo can just import/call functions in settings.py, instead having to duplicate this code every time you need to access settings, having to define `SETTINGS = 'settings.yaml'` everywhere, etc

Up to you if you want to do it here, or as a different ticket/PR

**Review by yhus — APPROVED** [review]

**Comment by yhus** [comment]
Good cleanup!

**Review by chunloy — APPROVED** [review]
🔥 awesome job!

**Review by tonyxuan2021 — APPROVED** [review]
Can't believe you fixed everything. Really good job Chao!

File: app.py (hunk 1 above)
Comment (StudentCZ) [reply app.py:14]: I looked up on how to do what you suggested and I will do it in a different PR. I will make a Jira ticket and put that in the backlog so I don't forget.

**Pull request merged**